weather_api.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import httpx
+from dotenv import load_dotenv
+from httpx import AsyncClient
+
+load_dotenv()
+
+URL = f"https://api.weatherapi.com/v1/current.json"
+API_KEY = os.getenv("API_KEY")
```

Comment:
Ensure that the `.env` file contains the `API_KEY` variable and that it is correctly loaded. If the `API_KEY` is `None`, the API request will fail.